src/test/java/ru/ayaz/UserTest.java

```diff
@@ -0,0 +1,54 @@
+package ru.ayaz;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by ayaz on 11/5/16.
+ */
+public class UserTest {
+
+    private User user;
+
+    @Before
+    public final void before(){ user = new User(); }
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void passingNull_setName() throws Exception {
```

Comment:
Yes, I didn't pay attention to test naming. TDD book's naming style: givenWhenThen.
For example: 
`@Test
public void givenLastTurnWasXWhenNextPlayerThenO()
{
ticTacToe.play(1, 1);
assertEquals('O', ticTacToe.nextPlayer());
}`